code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -187,15 +187,23 @@
 
 /obj/item/weapon/card/id/New()
 	..()
-	spawn(30)
-	if(istype(loc, /mob/living/carbon/human))
-		blood_type = loc:dna:b_type
-		dna_hash = loc:dna:unique_enzymes
-		fingerprint_hash = md5(loc:dna:uni_identity)
+
+	spawn(30) //AWFULNESS AHOY
+	if(src && ishuman(loc))

```

Comment:
but there ISN'T a 3 sec pass, because that spawn is used incorrectly.

Also, it it would be `qdel()`'d the check would still be 1 and if it would be `del()`'d the proc would've been killed already.